sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3684,6 +3684,15 @@ def _rsync_down(args) -> None:
         subprocess_utils.run_in_parallel(_rsync_down, parallel_args)
         return dict(zip(job_ids, local_log_dirs))
 
+    def sync_down_serve_logs(self, *args, **kwargs):
+        # yapf: disable
+        # pylint: disable=import-outside-toplevel
+        from sky.serve.serve_logs import sync_down_serve_logs
+
+        # pylint: enable=import-outside-toplevel
+        # yapf: enable
+        return sync_down_serve_logs(self, *args, **kwargs)
```

Comment:
Lets move it completely to serve core. Reference #4046

Reply:
Got it. Still working on it 🫡